tests/integration/tests/t05_simple_browsing.py

```diff
@@ -102,15 +102,15 @@ def __init__(self, *args, **kwargs):
         self.server_process = None
 
     @property
-    def tools(self) -> list[ToolSpec]:
+    def tools(self) -> list[Tool]:
         """List of tools available to the agent."""
         if self.cwd is None:
             raise ValueError("CWD must be set before accessing tools")
         register_tool("BashTool", BashTool)
         register_tool("FileEditorTool", FileEditorTool)
         return [
-            ToolSpec(name="BashTool", params={"working_dir": self.cwd}),
-            ToolSpec(name="FileEditorTool", params={"workspace_root": self.cwd}),
+            Tool(name="BashTool", params={"working_dir": self.cwd}),
+            Tool(name="FileEditorTool", params={"workspace_root": self.cwd}),
```

Comment:
@OpenHands, do NOT edit this PR, please checkout a new branch and start a new PR and remove all the patterns of `params={"working_dir"` and `params={"workspace_root"` for tools -- this is no longer supported